job_spec/RIVER_WIDTH.yml

```diff
@@ -41,9 +41,21 @@ RIVER_WIDTH:
         - --resolution
         - 10.0
         - Ref::granules
```

Comment:
We might want to explicitly write down all the RTC parameters here since we hard-coded them in the container before:
https://github.com/ASFHyP3/hyp3-gamma/pull/435/files#diff-b8e4c7543c71597a2c4ee2a5310294e9002965ca11118054a146d268748a4b6dL213-L225